lib/21-day-challenge-countdown.rb

```diff
@@ -9,21 +6,21 @@ class Countdown
     attr_reader :current_day
 
     def initialize
-      Time.zone = 'UTC'
-      start = ActiveSupport::TimeZone.new('Mountain Time (US & Canada)')
-        .parse('13/04/2015 12:00').to_datetime
-      now = Time.zone.now.to_datetime
-      elapsed = (now - start)
-      @current_day = (elapsed).ceil
-      next_deadline = start + @current_day
-      left_today = next_deadline.to_time - now.to_time
-      @time_left = Time.zone.at(left_today)
+      start_time = Time.parse('2015-05-04 12:00:00 -0600')
+      now = Time.now
+
+      elapsed = (now - start_time)
+      @current_day = (elapsed / 60/ 60 / 24).ceil
+      current_day_in_seconds = @current_day * 60 * 60 * 24
+
+      next_deadline = start_time + current_day_in_seconds - 1 # one second is substracted at the end of day
+      left_today = next_deadline - now
+      @time_left = Time.at(left_today).gmtime
       # left_string = @time_left.strftime("%H:%M:%S")
     end
 
     def deadline
-      "Today is day #{current_day}. You have #{time_left.strftime('%H:%M:%S')
-                                             } left to send your pull request."
+      "Today is day #{current_day}. You have #{time_left.strftime('%H')} hours, #{time_left.strftime('%M')} minutes and #{time_left.strftime('%S')} seconds left to send your pull request."
```

Comment:
@kotp What tests? I don't see any specs. Unless you mean 'hands testing by Alberto' in which case, yes. All returns I tested are the same as the gem's.

Re: .deadline's return, how could I accomplish that? It's a particular long string due to string interpolation.